### DISCUSSION TITLE
"Test Trained Model" step doesn't trigger "automatic1111 webui"

### DISCUSSION THREAD

**Discussion by davidfreitag2** [TheLastBen/fast-stable-diffusion#1818] [General]
How long should it take for the "Test Trained Model" step to run, with 10 images of 512x512. Message bar says "connected", but I don't get the "automatic1111 webui" to run a test.  It's been running about 2 hrs.
Thanks

**Comment by TheLastBen** [comment]
did you click on the link that showed above "connected" ?

**Comment by davidfreitag2** [comment]
Thanks for the quick reply. That does the trick!  It opened the ui.

Thanks!
